pkg/statsd/aggregator.go

```diff
@@ -170,6 +170,14 @@ func (a *MetricAggregator) Flush(flushInterval time.Duration) {
 			timer.PerSecond = 0
 		}
 	})
+
+	a.metricMap.Timers.Each(func(key, tagsKey string, timer gostatsd.Timer) {
+		histogram := LatencyHistogram(timer)
```

Comment:
This should calculate histograms, or aggregated functions (lines 78-169), not both.